criu/pagemap.c

```diff
@@ -416,7 +416,11 @@ static int maybe_read_page_img_cache(struct page_read *pr, unsigned long vaddr,
 	pr_debug("\tpr%lu-%u Read page from self %lx/%"PRIx64"\n", pr->img_id, pr->id, pr->cvaddr, pr->pi_off);
 	while (1) {
 		ret = read(fd, buf + curr, len - curr);
-		if (ret < 0) {
+		if (ret == 0) {
+			pr_err("Reached EOF unexpectedly while reading page from image\n");
+			return -1;
+		}
```

Comment:
Continuation (else if) of the statement should be on the same line as the closing bracket https://www.kernel.org/doc/html/v4.10/process/coding-style.html#placing-braces-and-spaces

Reply:
https://github.com/checkpoint-restore/criu/pull/1022

Reply:
https://github.com/checkpoint-restore/criu/pull/1022